modules/deploy/pages/deployment-option/cloud/gcp-private-service-connect.adoc

```diff
@@ -0,0 +1,321 @@
+- `BYOVPC_SUBNET_PODS_RANGE_NAME`: The name of the IPv4 range designated for K8s pods.
+- `BYOVPC_SUBNET_SERVICES_RANGE_NAME`: The name of the IPv4 range designated for services.
+- `BYOVPC_SUBNET_MASTER_RANGE`: The master IPv4 range.
+- `BYOVPC_PSC_NAT_SUBNET_NAME`: The name of the GCP subnet that was created for PSC NAT.
+- `BYOVPC_AGENT_SERVICE_ACC_EMAIL`: The email for the agent service account.
+- `BYOVPC_CONNECTORS_SERVICE_ACC_EMAIL`: The email for the connectors service account.
+- `BYOVPC_CONSOLE_SERVICE_ACC_EMAIL`: The email for the console service account.
+- `BYOVPC_REDPANDA_SERVICE_ACC_EMAIL`: The email for the Redpanda service account.
+- `BYOVPC_GKE_SERVICE_ACC_EMAIL`: The email for the GKE service account.
+- `BYOVPC_TIERED_STORAGE_BUCKET`: The name of the Google Storage bucket to use for tiered storage.
+--
++
+[,bash]
+----
+CLUSTER_POST_BODY=`cat << EOF
+{
+    "cloud_provider": "CLOUD_PROVIDER_GCP",
+    "connection_type": "CONNECTION_TYPE_PRIVATE",
+    "type": "TYPE_BYOC",
+    "name": "$CLUSTER_NAME",
+    "namespace_id": "$NAMESPACE_ID",
+    "network_id": "$NETWORK_ID",
+    "region": "$REGION",
+    "zones": $ZONES,
+    "throughput_tier": "$THROUGHPUT_TIER",
+    "redpanda_version": "$REDPANDA_VERSION",
+    "private_link": {
+        "enabled": true,
+        "gcp": {
+            "consumer_accept_list": $CONSUMER_ACCEPT_LIST
+        }
+    },
+    "customer_managed_resources": {
+        "gcp": {
+            "subnet": {
+                "name":"$BYOVPC_SUBNET_NAME",
+                "secondary_ipv4_range_pods": { "name": "$BYOVPC_SUBNET_PODS_RANGE_NAME" },
+                "secondary_ipv4_range_services": { "name": "$BYOVPC_SUBNET_SERVICES_RANGE_NAME" },
+                "k8s_master_ipv4_range": "$BYOVPC_SUBNET_MASTER_RANGE"
+            },
+            "psc_nat_subnet_name": "$BYOVPC_PSC_NAT_SUBNET_NAME"
+            "agent_service_account": { "email": "$BYOVPC_AGENT_SERVICE_ACC_EMAIL" },
+            "connector_service_account": { "email": "$BYOVPC_CONNECTORS_SERVICE_ACC_EMAIL" },
+            "console_service_account": { "email": "$BYOVPC_CONSOLE_SERVICE_ACC_EMAIL" },
+            "redpanda_cluster_service_account": { "email": "$BYOVPC_REDPANDA_SERVICE_ACC_EMAIL" },
+            "gke_service_account": { "email": "$BYOVPC_GKE_SERVICE_ACC_EMAIL" },
+            "tiered_storage_bucket": { "name" : "$BYOVPC_TIERED_STORAGE_BUCKET" },
+        }
+    }
+}
+EOF`
+
+curl -vv -X POST \
+-H "Content-Type: application/json" \
+-H "Authorization: Bearer $AUTH_TOKEN" \
+-d "$CLUSTER_POST_BODY" $PUBLIC_API_ENDPOINT/v1beta1/clusters
+----
+
+== Enable PSC on an existing BYOC cluster
+
+. In the Redpanda Cloud UI, go to the cluster overview and copy the cluster ID from the **Details** section.
++
+[,bash]
+----
+CLUSTER_ID=<cluster_id>
+----
+
+. Update the Redpanda Cloud Agent IAM role. This step is required only for clusters with customer-managed resources.
```

Comment:
Link "customer-managed resources" to https://deploy-preview-346--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/vpc-byo-gcp/?